treeherder/perf/stats.py

```diff
@@ -315,6 +293,58 @@ def interpret_effect_size(delta):
         return "large"
 
 
+def interpret_cles_direction(cles, pvalue_threshold=PVALUE_THRESHOLD):
+    greater_rev = None
+    if cles is None:
+        return "CLES cannot be interpreted", greater_rev
+    if cles > pvalue_threshold:
+        greater_rev = "base"
+        return f"{cles:.0%} chance a base value > a new value", greater_rev
+    if cles < pvalue_threshold:
```

Comment:
I see that you removed the `=` from the code that was here before. I think it would be good to add it back to this line. Otherwise, an equal will mean that it can't be interpreted.